src/test/unit/web_interface/test_plugin_routes.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask
+from flask_restful import Api
+from itertools import chain
+import os
+from unittest import TestCase
+
+from helperFunctions.config import get_config_for_testing
+from helperFunctions.fileSystem import get_src_dir
+from test.unit.web_interface.rest.conftest import decode_response
+
+from web_interface.components.plugin_routes import PluginRoutes, PLUGIN_CATEGORIES
+
+
+class PluginRoutesMock(PluginRoutes):
+    def __init__(self, app, config, api=None):
+        self._app = app
+        self._config = config
+        self._api = api
+
+
+class TestPluginRoutes(TestCase):
+
+    def setUp(self):
+        self.app = Flask(__name__)
+        self.app.config.from_object(__name__)
+        self.api = Api(self.app)
+        self.config = get_config_for_testing()
+
+    def test__get_modules_in_path(self):
+        plugin_dir_path = os.path.join(get_src_dir(), 'plugins')
+        plugin_folder_modules = PluginRoutes._get_modules_in_path(plugin_dir_path)
```

Comment:
Technically this is not a unit test since it depends on pkgutil functionality. But that's okay.